src/ExpandableSection.js

```diff
@@ -144,7 +187,7 @@ export default function ExpandableSection(props) {
   }
 
   const handleChange = withDefaultHandler(onChange, (e, expanded) => {
-    if (props.expanded == null) {
+    if (!props.expanded) {
```

Comment:
The logic behind it now is different than it was before. I think It should be as it was before.

Reply:
I see the difference now. I changed this to use `isNil`, since the coercive comparison can be confusing and always seems to just look like a typo.